packages/utilities/psammead-styles/src/fonts.js

```diff
@@ -1,8 +1,35 @@
 const baseFontUrl = 'https://gel.files.bbci.co.uk/r2.511/';
-const nassimPersianBaseUrl =
+
+const baseUrlBBCNassimArabic =
+  'https://ws-downloads.files.bbci.co.uk/fonts/NassimArabic/v1.441/';
+
+const baseUrlBBCNassimPashto =
+  'https://ws-downloads.files.bbci.co.uk/fonts/NassimPashto/v1.57/';
+
+const baseUrlBBCNassimPersian =
   'https://ws-downloads.files.bbci.co.uk/fonts/NassimPersian/v1.511/';
 
-// Serif
+const baseUrlBBCNassimUrdu =
+  'https://ws-downloads.files.bbci.co.uk/fonts/NassimUrdu/v1.60/';
+
+const baseUrlIskoolaPotaBBC =
+  'https://ws-downloads.files.bbci.co.uk/fonts/IskoolaPota/v5.91/';
+
+const baseUrlLatha = 'https://ws-downloads.files.bbci.co.uk/fonts/Latha/v5.90';
+
+const baseURlMallanna =
+  'https://ws-downloads.files.bbci.co.uk/fonts/Mallanna/v1.0.4/';
+
+const baseUrlNotoSansEthiopic =
+  'https://ws-downloads.files.bbci.co.uk/fonts/NotoSansEthiopic/v1.901/';
+
+const baseUrlPadauk =
+  'https://ws-downloads.files.bbci.co.uk/fonts/Padauk/v2.8/';
+
+const baseUrlShonarBangla =
+  'https://ws-downloads.files.bbci.co.uk/fonts/ShonarBangla/v5.91/';
+
```

Comment:
Would it be worth DRYing up the hostname for maintenance purposes?

Reply:
I'd prefer not to, since this way it's really easy to copy the url in your browser to check it works as expected with the relevant font file name.